schemas/com.snowplowanalytics.snowplow.badrows/schema_violations/jsonschema/1-0-0

```diff
@@ -0,0 +1,303 @@
+              "anyOf": [
+                {
+                  "description": "Schema is invalid and cannot be used to validate an instance",
+                  "properties": {
+                    "errors": {
+                      "type": "array",
+                      "minItems": 1,
+                      "items": {
+                        "properties": {
+                          "path": {
+                            "type": "string",
+                            "description": "JSON Path to an issue in the faulty JSON Schema"
+                          },
+                          "message": {
+                            "type": "string",
+                            "description": "Human-readable message describing the issue with the schema"
+                          }
+                        },
+                        "required": [ "path", "message" ],
+                        "additionalProperties": false
+                      }
+                    }
+                  },
+                  "required": [ "errors" ],
+                  "additionalProperties": false
+                },
+                {
+                  "description": "Invalid data - Schema was resolved and valid, but the given instance is not valid against it. The error comes from a validator",
+                  "properties": {
+                    "errors": {
+                      "type": "array",
+                      "minItems": 1,
+                      "items": {
+                        "properties": {
+                          "message": {
+                            "type": "string",
+                            "description": "Human-readable message describing the problem in the invalidated data"
+                          },
+                          "path": {
+                            "type": "string",
+                            "description": "JSON Path to an issue in the faulty data"
+                          },
+                          "keyword": {
+                            "type": "string",
+                            "description": "JSON Schema keyword that invalidated this instance"
+                          },
+                          "targets": {
+                            "type": "string",
+                            "description": "Violating value"
+                          }
+                        },
+                        "required": [ "message", "path", "keyword", "targets" ],
+                        "additionalProperties": false
+                      }
+                    }
+                  },
+                  "required": [ "errors" ],
+                  "additionalProperties": false
+                }
+              ]
+            }
+          ]
+        }
+      },
+      "required": [ "timestamp", "schemaKey", "messages" ],
+      "additionalProperties": false
+    },
+    "payload": {
```

Comment:
I think that's  an interesting choice;
So if I understand correctly,  the whole payload of the event is inlined here, which makes sence, so the shredded events are still searchable; for me this is perfect since I'd be still able to search for the app_id and sort ideally errors by `app_id` , `event_name` etc.

Reply:
I think this is part of the payload (e.g., the core event but not contexts / derived_contexts).

Reply:
yes that's right